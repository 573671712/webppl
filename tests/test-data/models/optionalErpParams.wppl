var makeCoin = function(p) {
  return Enumerate(function() {return flip(p);});
};

var FairCoin = makeCoin(.5);

var model = function() {
  var x = flip(0.5);
  var y = sample(FairCoin, x ? [0] : undefined);
  var z = sample(FairCoin, x ? undefined : [0]);
  return y && z;
};
