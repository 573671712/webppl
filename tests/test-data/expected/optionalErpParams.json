{
  "hist":  {
    "true": 0.25,
    "false": 0.75
  }
}
